omod/src/main/resources/config.xml

```diff
@@ -153,9 +153,9 @@
     </globalProperty>
 
     <globalProperty>
-        <property>ugandaemr.dsdm.conceptId</property>
-        <description>A Coded concept with all DSDM Concept</description>
-        <defaultValue>165143</defaultValue>
+        <property>ugandaemr.hts.recency</property>
```

Comment:
@echiteri You seem to have replaced the property ugandaemr.dsdm.conceptId instead of just adding ugandaemr.hts.recency

Reply:
@ssmusoke this is resolved